_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -144,7 +144,51 @@ software.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME:bitschmidty
+- **Blockstream announces LNsync:**
+  [LNsync][blockstream blog lnsync] allows Lightning wallets to more quickly sync the
+  LN topology in order to optimally route a payment to its destination. An open
+  source plugin, [historian][github historian], provides this functionality for
+  C-Lightning users.
+
+- **Rust light client Nakamoto released:**
+  Alexis Sellier [released Nakamoto][nakamoto blog], a "high-assurance Bitcoin
+  light-client implementation in Rust, with a focus on low resource utilization,
+  modularity and security".
+
+- **Blockstream Satellite broadcasting LN data, Bitcoin Core source:**
+  Blockstream's Satellite, in addition to broadcasting Bitcoin blockchain data,
+  now includes the [Bitcoin Core source code][blockstream satellite bitcoin
+  code], code for a satellite-optimized Bitcoin fork, [Bitcoin
+  Satellite][github bitcoin satellite], and [LN gossip data][blockstream
+  satellite ln data].
+
+- **Blockstream Green implements CSV:**
+  Green Wallet has previously used nLockTime as a wallet recovery mechanism,
+  requiring users to receive backup emails with pre-signed transactions from
+  Blockstream after each transaction in order to recover funds. By [implementing
+  an `OP_CHECKSEQUENCEVERIFY` (CSV) recovery mechanism][blockstream green csv],
+  wallets can be recovered without needing backup files or associating an email
```

Comment:
Maybe: s/backup files/special backup files/.  That may make it clearer that whatever usual backup stuff Green requires (e.g. an HD seed) is still needed.